components/server/src/authorization/authorizer.ts

```diff
@@ -553,6 +561,16 @@ async function getSubjectFromCtx(passed: Subject): Promise<SubjectId> {
             : undefined,
     });
     if (!authViaContext) {
+        if (!passedSubjectId) {
+            const err = new ApplicationError(ErrorCodes.PERMISSION_DENIED, `Cannot authorize request`);
+            log.error("Authorizer: Cannot authorize request: missing SubjectId", err, {
+                match,
+                ctxSubjectId,
+                ctxUserId,
+                passedUserId,
+            });
+            throw err;
+        }
```

Comment:
do we want to do something with `ctx-user-id-missing` here? I understood this would be an error case, or at least we want to know if there is a call where the subjectId is not set on the context.

Reply:
Oh ignore this ☝️ , I overlooked the metrics call on line 541.

Reply:
No, we don't. We report it as metric, but without the feature flag on, it's not an error.